client/js/utils.js

```diff
@@ -29,6 +43,27 @@ function forceFocus() {
 	input.trigger("click").focus();
 }
 
+function collapse() {
```

Comment:
(I know I said the opposite of this previously, so this is not for this PR...it's for the future)

If we get more than this, we will probably want to split them up out of `utils.js`, but not into just another file. Probably have a directory with a file per command. But as I say, not for this PR.